site/homepage/content/release-notes.md

```diff
@@ -16,6 +16,10 @@ description = "Information about changes in recent Hono releases. Includes new f
 ### Fixes & Enhancements
 
 * Documentation for versions up to 1.11 is removed as outdated.
+* Optimized handling of MQTT / AMQP protocol adapter shutdown.
+  On shutdown, the adapter no longer explicitly unregister Command Consumers and no longer sends `disconnectedTtdEvent`.
+  The sending of `connectedTtdEvent` / `disconnectedTtdEvent` is moved from adapter to Command Router.
+  Command Router config now also requires a `hono.messaging` configuration (in case AMQP messaging is used).
 
```

Comment:
I am not sure if the readme change is precisely enough.

Reply:
("explicitly unregister*s*")
I think it would be helpful to state the motivation for the behaviour change, why this is an "optimized handling" (preventing delays, errors; expecting that events are most often superfluous because of devices reconnecting).

Reply:
I have added more details. Could you take an other look?